src/routers/people.ts

```diff
@@ -8,24 +8,127 @@ import { StatusCode } from '@/models/statusCodes';
+ *              Associated attributes can be checked via /peopleCategories
+ * @apiName GetPeopleCategory
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ * @apiError (404) Person category not found
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all people from category <category>
+ * @apiSuccessExample /people/farmers Success-Response:
+  [
+    {
+      "personUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
+      "firstName": "Zachariah",
+      "paymentFrequency": "weekly",
+      "notes": "Brother of Moses",
+    },
+    {
+      "personUuid": "4b3f23a3-04c4-468f-bdf5-f189a34d9f69",
+      "firstName": "Zachariah",
+      "paymentFrequency": "monthly",
+      "notes": "Has a nice car",
+    }
+  ]
+ */
 router.get('/:category', async (req, res) => {
-  res.status(StatusCode.OK).send('Successfully retrieved all people of type <something>');
+  res.status(StatusCode.OK).send('Successfully retrieved all people of category <something>');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {get} /people/<category>
+ * @description Creates a new person in the specified category.
+ *              Returns the UUID created for that person.
+ * @apiName CreatePerson
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ * @apiError (404) Person not found in category <category>
+ *
+ * @apiSuccess (200) {String} Successfully retrieved person of category <category>
+ * @apiSuccessExample /people/farmers Success-Response:
+ * { personUuid: "1e167b81-d816-497b-8c0c-36f4d6b2fd33" }
+ */
 router.post('/:category', async (req, res) => {
-  res.status(StatusCode.CREATED).send('Successfully created new <type>');
+  res.status(StatusCode.OK).send('Successfully created new <category>');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {get} /people/<category>/<uuid>
+ * @description Gets a particular person
+ * @apiName GetPerson
+ *
+ * @apiError (401) Unauthorized - Must be admin
+ * @apiError (404) Person not found in category <category>
+ *
+ * @apiSuccess (200) {String} Successfully retrieved person
+ * @apiSuccessExample /people/farmers/1e167b81-d816-497b-8c0c-36f4d6b2fd33 Success-Response:
+   {
+     "personUuid": "1e167b81-d816-497b-8c0c-36f4d6b2fd33",
+     "firstName": "Zachariah",
+     "paymentFrequency": "weekly",
+     "notes": "Brother of Moses",
+   }
+ */
 router.get('/:category/:uuid', async (req, res) => {
-  res.status(StatusCode.OK).send('Successfully got <type>');
+  res.status(StatusCode.OK).send('Successfully got <category>');
 }, authorized(UserType.ADMIN));
 
+/**
+ * @api {put} /people/<category>/<uuid>
+ * @description Returns a specific person from a specific category
+ *              and all their associated attributes.
+ *              Parameters have no effect on this request.
+ *              Associated attributes can be checked via /peopleCategories
+ * @apiName UpdatePerson
+ *
+ * @apiParam {String} [<attribute>] An attribute of a person with its value.

```

Comment:
maybe mention in the apiParams to check the /peopeCategories api to see all the possible attributes

Reply:
Was mentioned in description

Reply:
Can you move that comment to the apiParam tag? :)